### ISSUE TITLE
question about the pretrain epoch

### ISSUE THREAD

**Issue by zwei-lin** [chaytonmin/Voxel-MAE#2]
Hi, 
    Thanks for the great work!
    I am confused about the training epoch of VoxelMAE. The pretrain epoch mentioned in the paper is 20 and 10 for Waymo and nuScenes. However, I find the epoch in config files is 30 for both Waymo and nuScenes. Do we just take the checkpoint of 20th epoch and load it to the model when fine-tuning?
    Look forward to your reply.

**Comment by chaytonmin** [comment]
> Hi, Thanks for the great work! I am confused about the training epoch of VoxelMAE. The pretrain epoch mentioned in the paper is 20 and 10 for Waymo and nuScenes. However, I find the epoch in config files is 30 for both Waymo and nuScenes. Do we just take the checkpoint of 20th epoch and load it to the model when fine-tuning? Look forward to your reply.

Yes!
I did not do a lot of experiments on Waymo and nuScenes as it is time-consuming to train the models.  Maybe you can get better results with another epoch of the pretraining model or with another masking ratio.  Besides, I uploaded the wrong version of the paper to the arxiv, please download the newest version in the arxiv without the results of nuScenes.

**Comment by zwei-lin** [comment]
Thanks for your reply!

**Comment by FrontierBreaker** [comment]
@zwei-lin Hi, have you reproduced the results on Nuscenes?
